### PR TITLE
Explore looped memory usage for cytomining/CytoTable#75

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds changes which were added as part of exploratory work for cytomining/CytoTable#75. I consider these to be in-progress, and in-need of later additions. Just the same, I'm making this PR as a way to achieve gradual progress here and enable flexibility when it comes to other changes in this repo.

Thanks in advance for any feedback you may have.

### CODE REVIEW

File: notebooks/cytotable-issue-75/cytotable_objgraph.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+# ---
+
+# # CytoTable object graph analysis
+#
+# This notebook explores how CytoTable objects operate. The work is related to [CytoTable#75](https://github.com/cytomining/CytoTable/issues/75).
+
+# +
+import gc
+import sys
+
```
Comment (gwaybio) [inline notebooks/cytotable-issue-75/cytotable_objgraph.py:24]:
```suggestion
import pathlib

```

Comment (gwaybio) [reply notebooks/cytotable-issue-75/cytotable_objgraph.py:24]: To avoid ImportError in this notebook - needs to be rerun as well

**Review by gwaybio — APPROVED** [review]
One ImportError to fix

I also noted a couple sporadic empty cells in many of the jupyter notebooks (often but not always the last cell).

I recommend going through all notebooks again (especially fixing the ImportError)

After you do this, there is no need for me to take another look

File: notebooks/cytotable-issue-75/cytotable_objgraph.py (hunk 1 above)
Comment (d33bs) [reply notebooks/cytotable-issue-75/cytotable_objgraph.py:24]: Thank you @gwaybio ! Added fixes for this.

**Comment by d33bs** [comment]
Thank you @gwaybio for this review, merging this in after some fixes.

**Pull request merged**